src/Media_Command.php

```diff
@@ -575,6 +575,8 @@ private function process_regeneration( $id, $skip_delete, $only_missing, $image_
 
 		$is_pdf = 'application/pdf' === get_post_mime_type( $id );
 
+		$original_meta = get_post_meta( $id, '_wp_attachment_metadata', true );
```

Comment:
Why not use `wp_get_attachment_metadata`?

Reply:
Good question ;-) I've updated the code.